terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
+  storage_type = "io1"
+  identifier = "fec-govcloud-prod"
```

Comment:
Ok, I think this might also be a good excuse to restore from a backup and document that. 

Let me see if I understand the steps:
change the code to allow destroy for this pr
double check that we have the last back up
work with infrastructure to run the terraform commands
Restore from back up (would we do this from the AWS console?)
make a new pr that disallows destroy 

doest that sound right @jmcarp